### ISSUE TITLE
Websocket support

### ISSUE THREAD

**Issue by joshhopkins** [luckymarmot/Paw-Issues#68]
Any plans on websocket support?

**Comment by sayduck-daniel** [comment]
and graphql subscriptions as well (based on websocket)

**Comment by marcfielding1** [comment]
Yeah I'd definitely use websocket and graphql support, most of our API's are on Lambda and we use web sockets quite a bit with mobile apps.

**Comment by joshhopkins** [comment]
@sayduck-daniel here's the [graphq issue](https://github.com/luckymarmot/Paw-Issues/issues/57)

**Comment by joeczucha** [comment]
Was this ever introduced? I found an [issue from May '17](https://stackoverflow.com/questions/34731795/websocket-request-426-upgrade-required) saying that it might be on the way so just wanted to check... thanks!

**Comment by jk** [comment]
Any news on websocket support?

**Comment by kazhuravlev** [comment]
+1

**Comment by navaru** [comment]
Meanwhile use: https://websocketking.com

**Comment by KuhlTime** [comment]
Not inside Paw either but at least sponsored by Paw [https://hoppscotch.io](https://hoppscotch.io) 🤷‍♂️

**Comment by marcfielding1** [comment]
Err, you do realize that there are people asking a question here that BUY a Paw license and would hope for some kind of response?

**Comment by vec715** [comment]
any updates?

**Comment by blevasseur-godaddy** [comment]
I'm curious if there's been any traction on this?

**Comment by marcfielding1** [comment]
use Postman, I removed my subscription for PAW because I'm not paying for a product that is quite clearly not supported properly - PAW WAS awesome in my book, but it's not the winner - also this is not a complicated feature, I don't quite understand why it's hard work engaging on it.

I mean common this was nearly a year ago?!

**Comment by marcfielding1** [comment]
It should be noted that this repo is being closed and there is a link on Readme for raising tickets though, so it might better to migrate our requests to here as this repo might not even be observed any more. Here is the link from the readme.

https://paw.cloud/support?app=com.luckymarmot.Paw

**Comment by alobaili** [comment]
> It should be noted that this repo is being closed and there is a link on Readme for raising tickets though, so it might better to migrate our requests to here as this repo might not even be observed any more. Here is the link from the readme.
> 
> 
> 
> https://paw.cloud/support?app=com.luckymarmot.Paw

That's not an excuse for the team not addressing the request and the many votes casted on it. I think giving an official statement on how existing issues will be handled before closing and abandoning the repo is the appropriate action.

**Comment by marcfielding1** [comment]
Whilst I do agree, during transition periods between orgs, things can get lost, now being annoyed is not going to change the facts, finding a solution will.

**Comment by marcfielding1** [comment]
We can troll this thread all year, not gonna make a difference, being angry only really changes your life - whilst we're entitled to feel aggrieved I don't believe the software owners want to lose subscribers, I believe they made a mistake, as have we all, so let's figure this out.

**Comment by marcfielding1** [comment]
To be blunt, if we could make this ourselves we wouldn't be here, so you have two choices, don't be here. Or engage. I think I'll select engage given I hadn't noticed the change of ownership.

**Comment by eivindml** [comment]
This would be a cool feature to include with Paw. 😊

**Comment by aiank** [comment]
Had the pro version so far, apparently which isn't required anymore. Web socket not yet present. Switching to **Firecamp** for good
